templates/environment.template.json

```diff
@@ -105,7 +105,7 @@
   },
   "variables": {
     "deploymentUrlBase": "https://raw.githubusercontent.com/SkillsFundingAgency/das-platform-building-blocks/master/templates/",
-    "sharedDeploymentUrlBase": "https://raw.githubusercontent.com/SkillsFundingAgency/das-shared-infrastructure/master/",
+    "sharedDeploymentUrlBase": "https://raw.githubusercontent.com/SkillsFundingAgency/das-shared-infrastructure/predeployment_logic/",
```

Comment:
Needs to go back to master before merge

Reply:
This is why you should be able to pass this in as a parameter when testing

Reply:
😂

Reply:
"out of scope"

Reply:
Open a PR to the README at https://github.com/SkillsFundingAgency/das-platform-building-blocks/ and suggest it

Reply:
fixed

@smneal Raise an RFC if you want to see that change implemented.

https://github.com/SkillsFundingAgency/das-platform-building-blocks/issues/new